File: Tareas_FdP/Tarea02vscode/Ejercicio11.py
print("Ingrese el largo: ")
largo = float(input())

print("Ingrese el ancho: ")
ancho = float(input())

print("Ingrese la altura: ")
altura = float(input())

area = 2 * ((largo * ancho) + (largo * altura) + (ancho * altura))
volumen = largo * ancho * altura

print("El área total es: ", area)
print("El volumen es: ", volumen)

File: Tareas_FdP/Tarea02vscode/Ejercicio11.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Ejercicio11 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        double largo, ancho, altura, area, volumen;

        System.out.println("Ingrese el largo: ");
        largo = Double.parseDouble(reader.readLine());

        System.out.println("Ingrese el ancho: ");
        ancho = Double.parseDouble(reader.readLine());

        System.out.println("Ingrese la altura: ");
        altura = Double.parseDouble(reader.readLine());

        area = 2 * ((largo * ancho) + (largo * altura) + (ancho * altura));
        volumen = largo * ancho * altura;

        System.out.println("El área total es: " + area);
        System.out.println("El volumen es: " + volumen);
    }
}